src/components/CollapsibleSection/CollapsibleSection.stories.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import CollapsibleSection from './CollapsibleSection'
+
+export default {
+  title: 'Components/CollapsibleSection',
+  component: CollapsibleSection,
+}
+
+const Template = (args) => <CollapsibleSection {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  description: 'this is a description',
```

Comment:
can we add an "Alternate" example here that uses a description with html elements too?

Reply:
definitely, thats a great idea